### PR TITLE
unmount: add page

### PR DESCRIPTION
**Pull request by Velox0**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/unmount.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# umount
+
+> The correct command is umount (u-mount).
+> Try `tldr umount` for the tldr.
+> More information: <https://manned.org/umount.8>.
```
Comment (vitorhcl) [inline pages/linux/unmount.md:5]:
```suggestion
> More information: <https://manned.org/umount.8>.

- View documentation for the right command:

`tldr umount`
```

File: pages/linux/unmount.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# umount
+
+> The correct command is umount (u-mount).
+> Try `tldr umount` for the tldr.
```
Comment (vitorhcl) [inline pages/linux/unmount.md:4]:
```suggestion
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
LGTM, thank you for your contribution! We generally follow the pattern I suggested

**Review by vitorhcl — APPROVED** [review]

**Comment by vitorhcl** [comment]
Thank you for your contribution and welcome to tldr! Let's see what the others think

**Comment by Velox0** [comment]
Thanks! I look forward to contributing more!

File: pages/linux/unmount.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# umount
+
+> The correct command is umount (u-mount).
+> More information: <https://manned.org/umount.8>.
+
+- View documentation for the right command:
```
[the remark this answers is not in the document]
Comment (Velox0) [reply pages/linux/unmount.md]: Thanks for your approval! I have commit the suggestion. Whenever it's convenient for you, could you please merge the pull request? Thanks!

**Review by cyqsimon — APPROVED** [review]

**Pull request merged**